cmd/make.go

```diff
@@ -116,7 +138,12 @@ func (cookedArgs cookedMakeCmdArgs) process() (err error) {
 			return err
 		}
 	case common.ELocation.File():
-		shareClient := common.CreateShareClient(cookedArgs.resourceURL.String(), credentialInfo, nil, options, to.Ptr(common.ETrailingDotOption.Disable()), &cookedArgs.resourceLocation)
+		var shareClient *share.Client
+		//nakulkarmerge - create a trailingdot client
+		shareClient, err = share.NewClientWithNoCredential(resourceURL, &share.ClientOptions{ClientOptions: options})
```

Comment:
as long as this is for only share operations, we should be fine without trailing dot options being set